### PR TITLE
lmao skill issue

### PR DESCRIPTION
**Pull request by mangorifo**
yes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,102 +2,117 @@
...
 </p>
 
+
 # GCAndroid
 
 Project is free to use. If you bought it from someone, please refund\
-When I see someone selling GCAndroid, then I will close the source code\
-Open source code will only be available for whoever affiliated with [Score Inc](https://github.com/Score-Inc)
+If I see someone selling GCAndroid, I will close the source code (or private)\
+Open source code will then only be available to those who are affiliated with [TheScore Inc](https://github.com/Score-Inc)
```
Comment (ElaXan) [inline README.md:10]: Fix your README Fix Issue

Change TheScore Inc to Score Inc

**Comment by akioukun** [comment]
Skill issue inc

File: README.md [hunk 2]
```diff
@@ -2,102 +2,117 @@
...
 apts=$(cat /etc/apt/sources.list); echo -e "$apts\ndeb http://ports.ubuntu.com/ubuntu-ports/ focal main restricted\ndeb http://ports.ubuntu.com/ubuntu-ports/ focal-updates main restricted\ndeb http://ports.ubuntu.com/ubuntu-ports/ focal universe" > /etc/apt/sources.list && apt update && apt install sudo
 ```
-4. Next is [installation](https://github.com/Score-Inc/GCAndroid#install)
+4. Next is [installation](https://github.com/Score-Inc/GCAndroid#install).
 
 
-## Install
-
+## Installation
+### MongoDB
```
Comment (ElaXan) [inline README.md:51]: Remove mongodb installation

**Comment by mangorifo** [comment]
> Skill issue inc

ikr

File: README.md [hunk 3]
```diff
@@ -2,102 +2,117 @@
...
 ```bash
 gcandroid
 ```
 
-## Edit Mongodb Database
+## Edit MongoDB Database
 
-To edit mongodb database, use [this app](https://play.google.com/store/apps/details?id=com.mongolime.app)\
-If you don't have money to buy for the app, you can search for other apk or search for crack App that exist in Internet...
+Note: All commands are executed inside Termux
```
Comment (ElaXan) [inline README.md:78]: Fix your issue, this not Termux, but other APK

File: README.md [hunk 4]
```diff
@@ -2,102 +2,117 @@
...
+You can make your own Private DNS to redirect to localhost.\
+And don't ask me, idk how lol\
+You can play without a Private DNS or Termux, but it needs Root/SuperUser access.\
+Do not ask me how to root, go to [XDA](https://www.xda-developers.com/) to find out how to root.
 
 ### Termux/mitmproxy
 
-You can use [my script](https://github.com/Score-Inc/AnimeGamePatch)\
-Select Menu on mitmproxy by running command `zex 4`\
+You may use [my script](https://github.com/Score-Inc/AnimeGamePatch) for this. To change it, enter the number `3`
```
Comment (ElaXan) [inline README.md:112]: Fix your issue, previous already correct

**Comment by mangorifo** [comment]
o

**Comment by akioukun** [comment]
so much skill issue

**Pull request closed without merging**